src/pycolmap/scene/camera.cc

```diff
@@ -110,30 +110,44 @@ void BindCamera(py::module& m) {
       .def("cam_from_img",
            &Camera::CamFromImg,
            "image_point"_a,
-           "Project point in image plane to camera frame.")
+           "Unproject point in image plane to camera frame.")
       .def(
           "cam_from_img",
           [](const Camera& self,
              const py::EigenDRef<const Eigen::MatrixX2d>& image_points) {
-            std::vector<Eigen::Vector2d> world_points(image_points.rows());
-            for (size_t idx = 0; idx < image_points.rows(); ++idx) {
-              world_points[idx] = self.CamFromImg(image_points.row(idx));
+            std::vector<Eigen::Vector3d> cam_rays(image_points.rows());
```

Comment:
Given that the return value type changed in this PR, I don't see a good way to overload this method to be backwards compatible. This will break users that depend on this function.